clients/venice-push-job/src/main/java/com/linkedin/venice/hadoop/AbstractVeniceMapper.java

```diff
@@ -135,37 +135,44 @@ protected boolean process(
     keyBW.set(recordKey, 0, recordKey.length);
     valueBW.set(finalRecordValue, 0, finalRecordValue.length);
 
-    // Compress based on all compression strategies to collect metrics if enabled.
-    if (compressionMetricCollectionEnabled) {
+    if (!compressionMetricCollectionEnabled) {
+      // collect the uncompressed value size
+      MRJobCounterHelper.incrTotalUncompressedValueSize(reporter, recordValue.length);
```

Comment:
We should collect this metric in all cases (helps with calculating compression ratio). I think the previous version of this code was more appropriate